pyre/bench/synth/binop_dunder_leaf_inline.py

```
@@ -0,0 +1,98 @@
+
+M = 1000000007
+# Sized so pypy's own execution clears `FLOOR_GATE_MIN_BASELINE_S`: below it
+# the floor gate declines the baseline as too small to judge and the ratio
+# reports startup rather than these loops.
+N = 1600000
+
+
+class Leaf:
+    __slots__ = ("x",)
+
+    def __init__(self, x):
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)